bin/run_tests

```diff
@@ -310,7 +310,7 @@ test_python_sdk() {
 
 test_burrow_evm() {
   run_docker_test ./families/burrow_evm/tests/unit_burrow_evm.yaml
-  run_docker_test burrow_evm_intkey_test
+  # run_docker_test burrow_evm_intkey_test
```

Comment:
This commit seems unnecessary in the context of this PR, since they are re-enabled later.